src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -203,24 +208,45 @@ export function getArNSMappingByInteractionType(
       break;
     case CONTRACT_TYPES.ANT: {
       switch (interactionType) {
-        case ANT_INTERACTION_TYPES.CREATE: {
-          const data = transactionData as CreateAntPayload;
+        case ANT_INTERACTION_TYPES.CREATE:
+          {
+            const data = transactionData as CreateAntPayload;
```

Comment:
you could us the `isObjectOfTransactionPayloadType<CreateAntPayload>` function here so you wouldn't even have to cast.

Reply:
added